actions/invite_actions.ts

```diff
@@ -3,19 +3,22 @@
 
 import * as TeamActions from 'mattermost-redux/actions/teams';
 import {getTeamMember} from 'mattermost-redux/selectors/entities/teams';
-import {TeamMemberWithError, TeamInviteWithError} from '@mattermost/types/teams';
 
-import {RelationOneToOne} from '@mattermost/types/utilities';
 import {ActionFunc, DispatchFunc, GetStateFunc} from 'mattermost-redux/types/actions';
-import {UserProfile} from '@mattermost/types/users';
+
 import {getChannelMembersInChannels} from 'mattermost-redux/selectors/entities/channels';
 import {joinChannel} from 'mattermost-redux/actions/channels';
-import {Channel, ChannelMembership} from '@mattermost/types/channels';
 
 import {addUsersToTeam} from 'actions/team_actions';
 import {t} from 'utils/i18n';
 import {localizeMessage} from 'utils/utils';
-import {isGuest} from 'mattermost-redux/utils/user_utils';
+import {isAdmin, isGuest} from 'mattermost-redux/utils/user_utils';
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
+
+import {Channel, ChannelMembership} from '@mattermost/types/channels';
+import {UserProfile} from '@mattermost/types/users';
+import {RelationOneToOne} from '@mattermost/types/utilities';
```

Comment:
Nit: Import order should be:
* place the `@mattermost/types` imports above the mattermost-redux imports: it is an exposed package and therefore more external.
* new mattermost-redux imports should be grouped together with existing ones, at the top of the file.

Reply:
hmm.. The IDE does the import arrangement for me on save. I have configured the IDE to use our .eslintrc.json on save but seems like its either missing some lint rules or using some extra rules. 
![Screenshot 2022-10-12 at 15 47 42](https://user-images.githubusercontent.com/841955/195346604-bb74c2e3-1f12-48fd-884a-35953a34d30e.png)